docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-feature-integration.md

```diff
@@ -1469,6 +1498,65 @@ class ShopApplicationDependencyProvider extends SprykerShopApplicationDependency
 }
 ```
 
+**src/Pyz/Yves/ProductDetailPage/Controller/ProductController.php**
```

Comment:
We do not add project-level changes to IG's, unless they are intended to be there (Project-only changes).